testprojects/src/java/org/pantsbuild/testproject/extra_jvm_options/BUILD

```diff
@@ -16,3 +16,9 @@ jvm_app(
     binary=':noopts',
     dependencies=[':noopts'],
 )
+
+from os.path import join
+python_binary(
+    name=join('python_app'),
```

Comment:
Why `join`? This is a no-op?

Reply:
No, that is a leftover. I will remove it tomorrow.